features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Scopes/Delete.jsx

```diff
@@ -122,7 +122,7 @@ function Delete(props) {
                     values={{ scope: scopeName }}
                 />}
                 labelOk={<FormattedMessage
-                    id='Apis.Details.Documents.Delete.document.listing.label.ok'
+                    id='Apis.Details.Documents.Delete.document.listing.label.ok.yes'
```

Comment:
id violating the agreed format

Reply:
There are two id's with "Apis.Details.Documents.Delete.document.listing.label.ok" for confirm and yes button. Hence, "yes" has been added to end of the id as confirm